runtime/rasdump/javadump.cpp

```diff
@@ -1776,6 +1782,55 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 		"NULL\n"
 		"NULL           ------------------------------------------------------------------------\n"
 	);
+#if defined(J9ZTPF)
+
```

Comment:
Please put the blank line before the new block.

Reply:
Apologies for delay, done!